projects/media-viewer/src/lib/annotations/comment-set/comment-set.component.spec.ts

```diff
@@ -267,7 +267,7 @@ describe('CommentSetComponent', () => {
 
     component.onSelect(annotationId);
 
-    expect(mockAnnotationService.selectAnnotation).toHaveBeenCalledWith(annotationId);
+    expect(mockAnnotationService.selectAnnotation).toHaveBeenCalled();
```

Comment:
is the annotationId not required anymore?

Reply:
I am not sure this was failing by default the one that has written it needs to go in and check why